.circleci/config.yml

```diff
@@ -17,11 +17,14 @@ jobs:
             export CA_BUNDLE="abcdefabcdef"
             export MESH_REGION=us-west-2
             export MESH_NAME=mesh
+            export VERSION=v1
```

Comment:
Why is this `v1` and not `latest` or `v0.1.6`?

Reply:
Its just a kube yaml syntax validation test, the contents don’t matter.